sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/HDFSMetadataLog.scala

```diff
@@ -114,12 +116,16 @@ class HDFSMetadataLog[T <: AnyRef : ClassTag](sparkSession: SparkSession, path:
     }
   }
 
-  /** Write a batch to a temp file then rename it to the batch file.
-   *
-   * There may be multiple [[HDFSMetadataLog]] using the same metadata path. Although it is not a
-   * valid behavior, we still need to prevent it from destroying the files.
-   */
   private def writeBatchToFile(metadata: T, path: Path): Unit = {
+    writeBatchToFileWithRetries(metadata, path, 1)
+  }
+
+  /** Write a batch to a temp file then rename it to the batch file.
+    *
```

Comment:
nit: indent, use the java style comment here.

Reply:
Thanks your remind.